src/hud/newhud.asm

```diff
@@ -256,38 +256,48 @@ NewHUDArea:
 RTS
 
 NewHUDItems:
-        LDA.w DashItemsCollected : BEQ .none
-                LDA.w #$0020 : LDX.w #HUDItemTiles+$00 ; PV
-                BIT.w DashItemsCollected : BEQ +
-                        INX #2
-                +
-                BIT.w DashItemsEquipped : BEQ +
-                        INX #2
-                +
-                LDA.w $0000,X : STA.l RightHUDThree+$02
+        LDX.w #HUDItemTiles+$00 ; PV
+        ; Is there a PV in the game?
+        LDA.l SeedFlags+$04 : BIT #$00C0 : BNE .skip_heat
```

Comment:
I wonder if we could use `HeatDamageTable` here to check for reduced damage and/or maybe a new flag for whether we want the icon to indicate reduced heat which the randomizer should be able to determine. I like the idea of SeedFlags being something like a manifest for the randomizer or other tools to read but not using it for game behavior if that makes sense. This is fine if you want to do that though, just a minor detail.

Reply:
Yea, that makes sense. I'll probably leave it for now, but I think the idea will be to put together a more permanent solution for after the playoffs. If it were a dedicated icon instead of using PV, we will probably just check the gravity heat reduction table as you suggested.